src/scss/_breakpoints.scss

```diff
@@ -2,7 +2,7 @@
 
 // 0px -> mobile
 @mixin mobile {
-  @media screen and (max-width: $breakpoint-small) {
+  @media screen and (max-width: $breakpoint-small - 1px) {
```

Comment:
Forgot to add this change in the previous PR that fixed sidebar scroll when `BaseLayout` height exceeds viewport height.